src/server/common/connectors/notify/notify.js

```diff
@@ -27,31 +28,28 @@ export async function sendNotification(data) {
   let response
 
   try {
-    response = await proxyFetch(NOTIFY_URL, {
-      method: 'POST',
-      body,
+    response = await Wreck.post(NOTIFY_URL, {
+      payload,
       headers: {
         Authorization: 'Bearer ' + createToken(notifyConfig.apiKey)
       },
-      signal: AbortSignal.timeout(timeout)
+      timeout
     })
   } catch (err) {
-    if (err.code && err.code === err.TIMEOUT_ERR) {
+    if (err.output?.statusCode === statusCodes.gatewayTimeout) {
       throw new Error(`Request to GOV.uk notify timed out after ${timeout}ms`)
+    } else if (err.data) {
+      const errors = JSON.parse(err.data.payload?.toString())
```

Comment:
I think this is good as is - in a 'let it crash' sort of way - but if payload is undefined, then so will `payload?.toString()` be. 

That'll cause a JSON.parse syntax error to be thrown.  In this case, I think that's what we want.  It'll be easy to get to the stack trace back to here and see what we want, and the user will see the outcome we intended